src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
+    const output = {};
+
+    // Append hidden element to body
+    const screenSizer = document.createElement('div');
+    screenSizer.classList.add('c-screen-sizer');
+
+    document.body.appendChild(screenSizer);
+
+    // It should have a 'content' property containing the breakpoints
+    const breakpoints = window.getComputedStyle(document.querySelector('.c-screen-sizer'))
+        .getPropertyValue('content')
```

Comment:
Can you update the PR description to include a list of the browsers tested, please? Asking as I don't think I've used this function before so want to be sure we aren't breaking anything 🦌

Reply:
Updated 👍 I was getting an error with `checkForUser is not a function` (from f-header), I just had to update the f-header version in globalweb to latest.